WordPress/WordPressTest/NotificationsServiceTests.swift

```diff
@@ -0,0 +1,179 @@
+import Foundation
+import XCTest
+import WordPress
+
+
+class NotificationsServiceTests : XCTestCase
+{
+    typealias StreamKind    = NotificationSettings.Stream.Kind
```

Comment:
I've been _not_ aligning stuff around, but sometimes it's stronger than me... since it's waaaay way more readable.
(Sorry about that!)